streams/src/main/java/org/apache/kafka/streams/kstream/internals/SessionWindowedKStreamImpl.java

```diff
@@ -196,11 +198,11 @@
                 retentionPeriod
             );
         }
-        final StoreBuilder<SessionStore<K, VR>> builder = Stores.sessionStoreBuilder(
+        final StoreBuilder<SessionStore<K, VR>> builder = new SessionStoreBuilder<>(
             supplier,
             materialized.keySerde(),
-            materialized.valueSerde()
-        );
+            materialized.valueSerde(),
+            Time.SYSTEM);
```

Comment:
With the new flusher/forwarder, do we need this still, or can we just roll this part of the diff back completely?

Reply:
Good call!